app/controllers/staff/activities_controller.rb

```diff
@@ -4,7 +4,7 @@ class Staff::ActivitiesController < Staff::BaseController
   include ActivityHelper
 
   def show
-    @activity = policy_scope(Activity).find(id)
+    @activity = Activity.find(id)
```

Comment:
Whilst I agree that `policy_scope` has probably been used in too many places, I don't think it should be removed everywhere.

In this example, if the user has a role that is allowed by `ActivityPolicy` to pass the `show?` check, then they can view *every* activity, even for organisations they don't belong to.

Not wrapping the query in `policy_scope` will skip applying the scope defined within the policy, which mostly defeats the object of using Pundit.

Reply:
I assume the Pundit defaults go on the basis of "If I can't see in on the index page, I can't reach the show page", but this feels insecure to me.